packages/app/src/views/auth/OAuthStart.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <div />
```

Comment:
is it possible to use </> instead of <div/>?
Would prevent rendering a linebreak (atleast I know this behaviour in React)